YelpAPI.podspec

```diff
@@ -26,9 +26,6 @@ Pod::Spec.new do |s|
   s.requires_arc = true
 
   s.source_files = 'Pod/Classes/**/*'
-  s.resource_bundles = {
-    'YelpAPI' => ['Pod/Assets/*.png']
-  }
 
   s.public_header_files = 'Pod/Classes/**/*.h'
```

Comment:
Oops, actually the source files and public_header_files need to be changed to not use 'Pod' anymore. Sorry!